server/src/weaverbird/pipeline/pipeline.py

```diff
@@ -184,6 +194,79 @@ def dict(self, *, exclude_none: bool = True, **kwargs) -> dict:
 ]
 
 
+def _remove_void_from_combo_condition(
+    condition: ConditionComboAnd | ConditionComboOr | None, combo: str
+) -> ConditionComboAnd | ConditionComboOr | None:
+    """
+    Loop into a combo condition and search for embedded condition that may
+    contains __VOID__ as column/value
+    """
+    if condition is None:
+        return None
+
+    loop_on: list[Condition] = []
+
+    for cond in getattr(condition, combo) or []:
+        if (transformed_condition := _remove_void_from_condition(cond)) is not None:
+            loop_on.append(transformed_condition)
+    if len(loop_on) == 0:
+        return None
+    else:
+        if combo == "and_":
+            condition.and_ = loop_on
+        else:
+            condition.or_ = loop_on
+
+    return condition
+
+
+def _remove_void_from_condition(condition: Condition | None) -> Condition | None:
```

Comment:
```suggestion
def _remove_void_from_condition(condition: Condition) -> Condition | None:
```

Same here, why should we receive `None` ?

Reply:
my code editor... in reality, there is no need for that None, removed !